### ISSUE TITLE
Access denied: Failed to execute 'open' on 'USBDevice'

### ISSUE THREAD

**Issue by baulusdev** [stacksmashing/gb-tetris-web#5]
Hi, I am getting this error when trying to connect my gameboy. I don't know where this error comes from.
I am using chromium on fedora linux. I tried the flatpak and native version. Both failed

It shows up in the permissions screen and lists as 'paired' but when clicking it, it gets stuck in the 'Connecting' screen and throws the following error in the js console.

Error:
`Uncaught (in promise) DOMException: Failed to execute 'open' on 'USBDevice': Access denied.`

![image](https://github.com/stacksmashing/gb-tetris-web/assets/82718849/e3ce1415-8012-46f8-a27c-72a87d08ffe0)

Not able to find any fix online. If this is the wrong place to ask, sorry to disturb.

**Comment by nezza** [comment]
Your user seems to not have acces so the USB device - you probably want to check your udev rules & user groups

**Comment by baulusdev** [comment]
Firstly: Thank you for your answer, you are correct. The user does not have permissions to the USB device.

Secondly: I followed [this article](https://developer.chrome.com/docs/apps/app_usb#caveats) in the chrome developer docs and [this forum post](https://superuser.com/questions/1210294/how-to-create-udev-rule-in-ubuntu-for-chrome-app-usb-access) but just can't get it to work.
Permissions are still only for root even after changing the config at `/etc/udev/rules.d/50-gameboy.rules` and rebooting udev and my machine multiple times.

![image](https://github.com/stacksmashing/gb-tetris-web/assets/82718849/6381b747-1f19-43d5-9c92-b520daa9671f)

![image](https://github.com/stacksmashing/gb-tetris-web/assets/82718849/03289fd1-44de-457f-89d3-c84c8d8fcae2)

I tried it on my windows partition and that worked totally fine (should have checked this earlier)

If you want I can close this issue, because it is not related to the code in this repo itself but my configuration not working.
